backend/supabase/seed.sql

```diff
@@ -58,23 +59,6 @@ create policy "public read" on users for
 select
   using (true);
 
-create index if not exists users_data_gin on users using GIN (data);
```

Comment:
Aren't other indexes for tables here in the seed.sql file? Currently, I would expect these indexes here. We really need to decide how we're organizing our sql.